Header header
string mac_addr
string dev_name
bool   is_present
int8   rssi
